### DISCUSSION TITLE
How to convert AIMD traces in XYZ format to json format files for fine-tuning

### DISCUSSION THREAD

**Discussion by ime1452** [CederGroupHub/chgnet#74] [Q&A]
I have some AIMD trace files XYZ generated by CP2K, and I tried to convert their format to json via ASE, but when I run `make_graphs.py`, I get an error!

```py
Importing: E:/chgnet-main/examples/pre-pos-1.json
Traceback (most recent call last):
  File "e:/chgnet-main/examples/make_graphs.py", line 101, in <module>
    data = StructureJsonData(data_path, graph_converter=converter)
  File "D:\Python38\lib\site-packages\chgnet\data\dataset.py", line 502, in __init__
    self.keys = [
  File "D:\Python38\lib\site-packages\chgnet\data\dataset.py", line 503, in <listcomp>
    (mp_id, graph_id) for mp_id, dic in self.data.items() for graph_id in dic
TypeError: 'int' object is not iterable
```

There seems to be a problem with the format converted by ASE, is there any other solution?

**Comment by janosh** [comment]
Can you change the package code and insert some print statements for debugging? e.g. add `print(f"{self.data[0]=}")` just before the line that throws the error. Alternatively, share one of your XYZ files so we can repro.

**Reply by ime1452** [reply]
AIMD.xyz is the original trajectory file (NVT,300 K) with cell parameters a=b=c=3.5nm. AIMD.json is converted by ASE.
[AIMD.zip](https://github.com/CederGroupHub/chgnet/files/12707619/AIMD.zip)
